test/test.js

```diff
@@ -1074,6 +1074,18 @@ describe('Page', function() {
       expect(element).toBe(null);
     }));
   });
+  describe('Page.$$', function() {
+    it('should query existing elements', SX(async function() {
+      await page.setContent('<div>A</div><br/><div>B</div>');
+      let elements = await page.$$('div');
+      expect(elements.length).toBe(2);
```

Comment:
Let's dump the content, so we at least know that it is creating ElementHandles.
expect(await Promise.all(elements.map(e => e.evaluate(g => g.textContent)))).toEqual(['A','B'])